File: 240610/숫자들의 배수/multiple-of-numbers.py
N = int(input())

fives = []
if N == 5 or N == 10:
    fives.append(N)

idx = 1
n_lst = [N]
while len(fives) != 2:
    idx += 1
    if (N * idx) % 5 == 0:
        fives.append(N * idx) 
    n_lst.append(N * idx)

print(*n_lst)

File: 251115/숫자들의 배수/multiple-of-numbers.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() 
{
    long long N;
    cin >> N;

    vector<long long> fives;
    if (N == 5 || N == 10) 
        fives.push_back(N);

    long long idx = 1;
    vector<long long> n_lst;
    n_lst.push_back(N);

    while (fives.size() != 2) 
    {
        idx++;
        long long val = N * idx;
        if (val % 5 == 0) 
            fives.push_back(val);
        n_lst.push_back(val);
    }

    for (size_t i = 0; i < n_lst.size(); i++) 
    {
        if (i > 0) cout << ' ';
        cout << n_lst[i];
    }
    cout << '\n';

    return 0;
}
